app_base.htm

```diff
@@ -0,0 +1,82 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="menu">
+    <nav class="navbar-inverse navbar-static-top">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="/index/"><i class="fa fa-music" aria-hidden="true"></i>   FOMO</a>
+          </div>
+          <ul class="nav navbar-nav">
+            <li class="${ 'active' if request.dmp.page == 'index' and request.dmp.app == 'homepage' else '' }"><a href="/index/">Home</a></li>
+            <li class="${ 'active' if request.dmp.page == 'sections' else '' }"><a href="/sections/">Sections</a></li>
+            <li class="${ 'active' if request.dmp.page == 'about' else '' }"><a href="/about/">About</a></li>
+            <li class="${ 'active' if request.dmp.page == 'contact' else '' }"><a href="/contact/">Contact</a></li>
+            <li class="${ 'active' if request.dmp.page == 'terms' else '' }"><a href="/terms/">Terms</a></li>
+            <li class="${ 'active' if request.dmp.page == 'faq' else '' }"><a href="/faq/">FAQ</a></li>
+            <li class="${ 'active' if request.dmp.app == 'catalog' else '' }"><a href="/catalog/">Products</a></li>
```

Comment:
hey so since this is only in your catalog app, you can take out all those if statements and just set the Products link to be active

Reply:
That's so true! Thanks lindsthemormon!